rosbag2_transport/test/rosbag2_transport/mock_sequential_reader.hpp

```diff
@@ -85,6 +85,11 @@ class MockSequentialReader : public rosbag2_cpp::reader_interfaces::BaseReaderIn
     num_read_ = 0;
   }
 
+  void add_event_callbacks(rosbag2_cpp::bag_events::ReaderEventCallbacks & /* callbacks */)
+  {
+    // TODO(gbiggs): Figure out how to mock this appropriately
```

Comment:
I think we can do the similar as in regular sequential writer  
i.e. 
```
 if (callbacks.write_split_callback) {
    callback_manager_.add_event_callback(
      callbacks.write_split_callback,
      bag_events::BagEvent::WRITE_SPLIT);
  }
```
And call 
```
  file_number++;
  auto info = std::make_shared<bag_events::WriteSplitInfo>();
  info->closed_file = "ClosedFile" + std::to_string(file_number);
  info->opened_file = "OpenedFile" + std::to_string(file_number);
  callback_manager_.execute_callbacks(bag_events::BagEvent::WRITE_SPLIT, info);
```
in every `N` call of mocked write method.

Reply:
Done in 71c5349. Thanks for the tips.